index.html

```diff
@@ -1,13 +1,342 @@
-    
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;800&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <nav class="nav-header">
+          <div class="logo">
+            <a href="#">My Logo</a>
+          </div>
+          <ul class="nav-menu">
+            <li><a href="#cards">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+          <div class="icon">
+            <i class="fa fa-bars" aria-hidden="true"></i>
+          </div>
+        </nav>
+      </header>
+      <section class="section_intro">
+        <div class="full_info">
+          <h1>
+            I’m Mahram Hossain <br />
+            Glad to see you!
+          </h1>
+          <div class="bio">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+
+          <div class="link">
+            <a href="#"> LET’S CONNECT </a>
+          </div>
+          <div class="social_btn">
+            <ul class="social_list">
+              <li class="social_item">
+                <a href="#"><img src="./images/twicon.svg" alt="twitter" /></a>
+              </li>
+              <li class="social_item">
+                <a href="#"
+                  ><img src="./images/Linkedin-icon.svg" alt="linkdin"
+                /></a>
+              </li>
+              <li class="social_item">
+                <a href="#"><img src="./images/micon.svg" alt="m pic" /></a>
+              </li>
+              <li class="social_item">
+                <a href="#"
+                  ><img src="./images/githubicon.svg" alt="github"
+                /></a>
+              </li>
+              <li class="social_item">
+                <a href="#"><img src="./images/peaceicon.svg" alt="peace" /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <!-- Start cards section -->
+
+      <section class="cards_section" id="cards">
+        <ul class="cards_list">
+          <li class="list_item mt">
```

Comment:
- Great Job implementing the project cards on desktop screen 👍🏽 
- I'd like to kindly suggest that we stick to the design on Figma. Can we fix the background color? You have used a shade that's darker than what's on Figma.

Here is what I mean 👇🏽 
 
| The background on your Design  | Background on Figma Design |
| :---:   | :---: | 
| <img width="767" alt="Screenshot 2023-04-07 at 23 51 21" src="https://user-images.githubusercontent.com/99246359/230704074-9caea447-88b5-4371-9414-ab84a1633508.png"> |<img width="547" alt="Screenshot 2023-04-08 at 00 01 37" src="https://user-images.githubusercontent.com/99246359/230704103-dc332fec-5464-4952-8474-d1e4b0508c7f.png">  |